### ISSUE TITLE
input event problems

### ISSUE THREAD

**Issue by ChALkeR** [mdn/content#2249]
# Request type

- [ ] Please close this issue, I accidentally submitted it without adding any details
- [ ] New documentation
- [x] Correction or update

# Details

On page [Web/Events/input](https://developer.mozilla.org/en-US/docs/Web/Events/input), MDN states that `input` event is implemented in Firefox, but actually that implementation differs from the standard.

It's documented in spec and MDN to have `.composed === true` set for `input` events, which looks to be true for Chrome 72 and Safari, but Firefox 65 has `.composed === false` for `input` events coming from a `<select>` element (and, possibly, some other elements).

**Upd**: _This seems to affect `<select>` and `<input type="checkbox">` at least, in Firefox 65. This does not seem to affect `<input type="text">` elements — they emit `input` events with `.composed` set to `true` in Firefox 65._

Here is a testcase, could be also [viewed on codepen](https://codepen.io/anon/pen/OdBMoG):
```js
<select>
  <option>change me</option>
  <option>select me</option>
</select>
<script>
  document.body.querySelector('select').oninput = e => {
    alert(`This should be true by spec: ${e.composed}`)
  }
</script>
```

Is something like this supposed to be mentioned in known issues on that MDN page, like other browser differences are?

Other browsers results (for `<select>` element):
* Chrome 72 on Linux, Android, and macOS report `true`.
* Safari on macOS reports `true`.
* Firefox 65 on Linux and macOS reports `false`, which seems to be invalid.
* Chrome 71 on macOS reports `false`, which also seems to be invalid.
* Firefox Nightly (67) reports `false`.

Corresponding spec links:
* https://w3c.github.io/uievents/#event-type-input
* https://www.w3.org/TR/uievents/#event-type-input
* https://dom.spec.whatwg.org/#event

MDN URL: https://developer.mozilla.org/en-US/docs/Web/Events/input

**Comment by ChALkeR** [comment]
This still holds, FF behavior differs from Chromium and conflicts with what's documented on MDN.
This looks like "Partial support", I think.

**Comment by ChALkeR** [comment]
> The input event fires when the value of an [`<input>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input), [`<select>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/select), or [`<textarea>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea) element has been changed.

This is also still wrong.

**Comment by wbamberg** [comment]
Yes, we had a rogue gitbot closing some issues that should have stayed open. It's a good reminder to fix some of these old issues though!
